config/config.go

```diff
@@ -2,3 +2,30 @@
 // Basic configuration like consul credentials and address, http port to listen for requests,
 // postgres schema name, credentials, and client timeout are read from environment variables.
 package config
+
+import (
+	"strings"
+
+	"github.com/lvl484/user-manager/logger"
+	"github.com/spf13/viper"
+)
+
+// NewPostgresConfig returns pointer to PointerConfig with data read from viper.config.json
+func NewLoggerConfig(configName, configPath string) (*logger.LogConfig, error) {
+	v := viper.New()
+	v.AddConfigPath(configPath)
+	v.SetConfigName(configName)
+	v.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
```

Comment:
Would you call that for every new func for getting configs?

Reply:
Fixed,  thanks for prompt